packages/terra-clinical-detail-view/src/DetailList.jsx

```diff
@@ -21,27 +21,53 @@ const propTypes = {
     PropTypes.objectOf(DetailListItem),
     PropTypes.arrayOf(PropTypes.objectOf(DetailListItem)),
   ]),
+
+  /**
+   * The list type based on whether the list items are single HTML elements or label-value pairs.
+   * Label-value pairs should be provided using the `<dt>` element for the label and the `<dd>` element for the value.
+   * The `terra-clinical-label-value-view` component with `isChildOfDescriptionList={true}` can also be used as it will
+   * return the label and value using the appropriate `<dt>` and `<dd>` elements.
```

Comment:
Can we add a new line here?

Reply:
Added here: 62650031643f4a37281e54a40d36d6c6507751e8